README.md

```diff
@@ -0,0 +1,6 @@
+This is the top level project for the Scala ITR. It will contain all the exercises and the test suite.
+
+**How to run test**
+
+Tests can be run from within labs directory like this:
+    sbt 'testOnly org.scalalabs.basic.lab01.HelloWorldExerciseTest'
```

Comment:
I agree it is useful to document this.

Perhaps we should recommend people to start sbt 'interactively' instead of starting sbt each time you want to run a test, though?

Reply:
good point. Maybe it's okay to add this as additional note and leave the line as it is?

Reply:
That's up to you, it will be an improvement in any case!

Reply:
okay I added recommendation note with link to official documentation. please check